server/src-rsr/initialise.sql

```diff
@@ -48,6 +48,17 @@ CREATE TABLE hdb_catalog.hdb_relationship
     FOREIGN KEY (table_schema, table_name) REFERENCES hdb_catalog.hdb_table(table_schema, table_name) ON UPDATE CASCADE
 );
 
+CREATE TABLE hdb_catalog.hdb_remote_relationship
+(
+   name TEXT NOT NULL,
+   table_schema TEXT NOT NULL,
+   table_name TEXT NOT NULL,
+   remote_schema TEXT NOT NULL,
+   remote_field TEXT NOT NULL,
```

Comment:
Let's use the following schema?

```
name text not null,
table_schema text not null, 
table_name text not null,
remote_schema text not null,
configuration jsonb not null
```

with unique key : `(name, table_schema, table_name)`